gadgets/src/bits/rca.rs

```diff
@@ -33,7 +33,7 @@ where
 // Generic impl
 impl<F: Field> RippleCarryAdder<F> for Vec<Boolean> {
     fn add_bits<CS: ConstraintSystem<F>>(&self, mut cs: CS, other: &Self) -> Result<Vec<Boolean>, SynthesisError> {
-        let mut result = Vec::with_capacity(self.len() + 1);
+        let mut result = Vec::with_capacity(self.len() + other.len() + 1);
```

Comment:
Actually I'll need to revert this one bit; I seem to have misread the `zip` as `chain`.

Reply:
Good catch, no problem!

Reply:
Fixed :ok_hand: